python/cudf/cudf/core/column/lists.py

```diff
@@ -285,3 +285,56 @@ def take(self, lists_indices):
             raise
         else:
             return res
+
+    def sort_values(
+        self,
+        ascending=True,
+        inplace=False,
+        kind="quicksort",
+        na_position="last",
+        ignore_index=False,
+    ):
+        """
+        Sort each list by the values.
+
+        Sort the lists in ascending or descending order by some criterion.
+
+        Parameters
+        ----------
+        ascending : bool, default True
+            If True, sort values in ascending order, otherwise descending.
+        na_position : {'first', 'last'}, default 'last'
+            'first' puts nulls at the beginning, 'last' puts nulls at the end.
+        ignore_index : bool, default False
+            If True, the resulting axis will be labeled 0, 1, ..., n - 1.
+
+        Returns
+        -------
+        ListColumn with each list sorted
+
+        Notes
+        -----
+        Difference from pandas:
+          * Not supporting: `inplace`, `kind`
```

Comment:
Question: should `inplace` be supported by passing into `_return_or_inplace`?

Reply:
If it's a one liner then sure, otherwise if it requires any implementation work lets just leave it unsupported for now and we can address if/when there's feedback to do so.

Reply:
It turns out it's more than a single line because `_return_or_inplace` internal does not handle `inplace` and `retain_index` simultaneously.